deploy/start.go

```diff
@@ -198,9 +194,33 @@ func RunTests(name string, ts []T) {
 			continue
 		}
 
-		rs[i] = RunStatsAvg(runs)
+		s, err := AverageStats(runs...)
+		if err != nil {
+			dbg.Fatal("Could not average stats for test ", i)
+		}
+		rs[i] = s
+		rs[i].WriteTo(f)
+		// Write the header if you still havent done it
+		if !headerWritten {
```

Comment:
Writing the header just seems odd in the loop - can't you take it out and decide upon the ```t T``` which header you have to write? Then the file-open and defer come out of the loop.